src/devices/machine/e05a30.cpp

```diff
@@ -167,6 +168,13 @@ void e05a30_device::write(offs_t offset, uint8_t data)
 	LOG("%s: e05a30_w([0xC0%02x]): %02x\n", machine().describe_context(), offset, data);
 
 	switch (offset) {
+	case 0x0:
+	case 0x1:
+	case 0x2:
+	case 0x3:
```

Comment:
The rest of this `switch` statement has two-digit hex case constants.  Please be consistent with it.